frontend/src/metabase/admin/performance/constants/simple.ts

```diff
@@ -0,0 +1,2 @@
+export const rootId = 0;
+export const defaultMinDurationMs = 1000;
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 3]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->These are in a separate file to avoid circular imports<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/43122?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->